commcare_connect/opportunity/views.py

```diff
@@ -1359,3 +1365,15 @@ def resend_user_invite(request, org_slug, opp_id, pk):
         invite_user.delay(user.id, access.pk)
 
     return HttpResponse("The invitation has been successfully resent to the user.")
+
+
+def sync_deliver_units(request, org_slug, opp_id):
+    status = status = HTTPStatus.OK
+    message = "Delivery unit sync completed."
+    try:
+        create_learn_modules_and_deliver_units.apply(args=[opp_id], throw=True)
+    except AppNoBuildException:
+        status = HTTPStatus.BAD_REQUEST
+        message = "No new updates available"
```

Comment:
It might be worth having a slightly more honest message here, like "Failed to retrieve updates" rather than none available.